release/README.md

```diff
@@ -0,0 +1,3 @@
+# Flutter Release Docker Image
+
+The release folder contains the Dockerfile which is used to create the docker image used by Louhi workflows for 3P releases. 
```

Comment:
I think this is why the build is failing, you've got a trailing space on this line:
```suggestion
The release folder contains the Dockerfile which is used to create the docker image used by Louhi workflows for 3P releases.
```

Reply:
Oh let me remove it, the `dart analyze` didn't show it.

Reply:
That solved the analyze step, it failing at the app_dart step. 

Looking into that.